### PR TITLE
Updates to the readme

### PR DESCRIPTION
**Pull request by danielle-casella-adams**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # composite-run-steps-action-template
 
 This template can be used to quickly start a new custom composite-run-steps action repository.  Click the `Use this template` button at the top to get started.
-
+- [Inputs](#inputs)
+- [Outputs](#outputs)
+- [Example](#example)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
```
Comment (JosephDSchwartz) [inline README.md:8]: Just out of curiosity, why was `Incrementing the Version` left off of this list?

**Review by hpractv — APPROVED** [review]
👍🏽

File: README.md (hunk 1 above)
Comment (JosephDSchwartz) [reply README.md:8]: I see it on the Javascript action template so now I'm thinking it was maybe a miss?

Comment (danielle-casella-adams) [reply README.md]: Because I generated the ToC before I copied that section in. :(

**Review by JosephDSchwartz — APPROVED** [review]

**Pull request merged**